app-libs/stf/src/lib.rs

```diff
@@ -332,24 +279,4 @@ mod tests {
 
 		assert!(signed_call.verify_signature(&mrenclave, &shard));
 	}
-
-	#[test]
-	fn new_payload_works() {
```

Comment:
Good catch - maybe this was an oversight?

Reply:
New `Payload` does no longer compile to std because the `SgxExternalitiesDiff` type misses the `Encode` and `Decode` trait bounds there. So this test can't be run in with cargo test. 

My opinion is that the constructor is implicitly tested in every other test involving `StatePayload` so I thought it is safe to remove that one. An enclave test explicitly for this I deemed an onverkill. 😄

Reply:
I see. I can agree to that. But I still would love to hear the profis opinion here @mullefel  :-)

Reply:
I think I can live with that, yes 👍